Framework/PythonInterface/plugins/algorithms/LinkedUBs.py

```diff
@@ -0,0 +1,380 @@
+from mantid.api import (DataProcessorAlgorithm, mtd, AlgorithmFactory,
```

Comment:
Really minor but it looks like we are missing the [license agreement](http://developer.mantidproject.org/Standards/PythonStandards.html#license) from this file.